README.md

```diff
@@ -44,26 +44,27 @@ Note that `bigint` types don't have a default `toJSON` method, so the above assu
+| bigIntType            |           `"bigint" \| "string" \| "number"`           |                                 `"bigint"`                                 | The type to use for BigInt model fields.                                                                                                                                                              |
+| decimalType           |          `"Decimal" \| "string" \| "number"`           |                                `"Decimal"`                                 | The type to use for Decimal model fields. The `Decimal` type here is just an interface with a `valueOf()` function. You will need to cast to an actual Decimal type if you want to use other methods. |
+| bytesType             | `"Buffer" \| "BufferObject" \| "string" \| "number[]"` |                                 `"Buffer"`                                 | The type to use for Bytes model fields. `BufferObject` is a type definition which matches the output of `Buffer.toJSON()`, which is called when running `JSON.stringify()` on a Buffer.               |
+| optionalRelations     |                       `boolean`                        |                                   `true`                                   | Controls whether model relation fields are optional. If `true`, all model relation fields will use `?:` in the field definition.                                                                      |
+| omitRelations         |                       `boolean`                        |                                  `false`                                   | Controls whether model relation fields are omitted. If `true`, model definitions will not include their relations.                                                                                    |
+| optionalNullables     |                       `boolean`                        |                                  `false`                                   | Controls whether nullable fields are optional. Nullable fields are always defined with `\| null` in their type definition, but if this is `true`, they will also use `?:`.                            |
+| prettier              |                       `boolean`                        |                                  `false`                                   | Formats the output using Prettier. Setting this to `true` requires that the `prettier` package is available.                                                                                          |
+| resolvePrettierConfig |                       `boolean`                        |                                  `false`                                   | Tries to find and use prettier config file in current project                                                                                                                                         |
```

Comment:
Technically this tries to find the config file relative to the output, which isn't necessarily in the current project. Maybe change the description to "Tries to find and use a Prettier config file relative to the output location."

Reply:
Makes sense